src/types/Option.ts

```diff
@@ -1,6 +1,6 @@
 import type { ReactNode } from 'react';
 
-export interface IOption<V extends string | number = string | number> {
+export interface Option<V extends string | number = string | number> {
   icon?: ReactNode;
```

Comment:
Can we change this to be `SelectOption` to be more specific. It is a pretty widely used type and `Option` isn't very specific.